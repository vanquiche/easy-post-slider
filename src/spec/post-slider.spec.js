import {
	getEditedPostContent,
	insertBlock,
	enablePageDialogAccept,
	createNewPost,
} from '@wordpress/e2e-test-utils';

describe( 'Post Slider Block', () => {
	beforeAll( async () => {
		await enablePageDialogAccept();
	} );

	beforeEach( async () => {
		await createNewPost();
	} );

	it( 'Post Slider block should be available', async () => {
		await insertBlock( 'Parfait Designs Post Slider' );
		expect(
			await page.$( '[data-type="parfait-designs/post-slider"]' )
		).not.toBeNull();
		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
