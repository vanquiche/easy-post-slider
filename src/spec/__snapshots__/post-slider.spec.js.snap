// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Slider Block Post Slider block should be available 1`] = `"<!-- wp:parfait-designs/post-slider /-->"`;
